### DISCUSSION TITLE
Confirmed working on...

### DISCUSSION THREAD

**Discussion by 007revad** [007revad/ContainerManager_for_all_armv8#5] [General]
Comment here with your Synology model and DSM version so I can update the [Confirmed working on](https://github.com/007revad/ContainerManager_for_all_armv8#confirmed-working-on) list.

**Comment by marmurek** [comment]
Working on DS218 @ DSM 7.2.1-69057 Update 3

**Comment by marcocarraro** [comment]
Message from terminal:
ContainerManager_for_all_armv8 v1.2.4
DS218j-j DSM 7.2.1-69057-4 
synology_armada38x_ds218j

You don't need this script. Container Manager is available
in Package Center for your synology_armada38x_ds218j

Unfortunately the package is not present in Package Center instead.

**Reply by 007revad** [reply]
@marcocarraro  Do you want to try a version of the script continues past that message?  https://github.com/007revad/ContainerManager_for_all_armv8/releases/tag/v1.3.5 

**EDIT** Does the following command return `x86_64` or `armv8` or something else?
```
uname -m
```

**Reply by marcocarraro** [reply]
Hi, thanks for the feedback, the command gives me:
armv7l

**Reply by marcocarraro** [reply]
And here is the error message running the v1.3.5
_ContainerManager_for_all_armv8 v1.3.5
DS218j-j DSM 7.2.1-69057-4 
synology_armada38x_ds218j

You don't need this script. Container Manager is available
in Package Center for your synology_armada38x_ds218j

ContainerManager is not installed

Line 127: ERROR Installing ContainerManager on /volume1 failed!
progstatus called from package_install line 247
Restoring synoinfo.conf_

**Reply by 007revad** [reply]
> the command gives me:
armv7l

After a bit of research I've found that the DS218j has a 32 bit CPU. Container Manager only installs on Synology NAS with a 64 bit CPU.

Marvell have some confusing CPU architecture naming:
- armada38x is 32 bit armv7
- armada37xx is 64 bit armv8

So Container Manager won't install or run on the following models which have a 32 bit CPU:
```
DS419slim
DS218j
RS217
RS816
DS416slim
DS416j
DS216j
DS216
DS116
```

**Reply by marcocarraro** [reply]
I can "read through the lines" it is not working and it never will, right? :_(

**Reply by 007revad** [reply]
Correct, unfortunately.

**Comment by Dchris832** [comment]
Working on DS218 @7.2.1-69057 Update 5

**Comment by Maxim-Mestdagh** [comment]
Working on DS218play with DSM 7.2.1-69057 Update 5, thanks you

**Comment by Grafyan** [comment]
Working on DS418j with DSM 7.2.1-69057 Update 5, thank you for your work :D

**Comment by Aitors30** [comment]
Hi, Installed in DS218Play but not Starting. DSM 7.2.2-72806. I don't know if the new 7.2.2 version affect to the Container packet.

Thanks in advance.

**Reply by Aitors30** [reply]
Ok, autoreply.... i need to be a Root user to do. forgot to put "sudo -i" before execute the script. Now it's work.

**Reply by Plantacom** [reply]
Hi. After login to root user the command line shows: _root@'yourDSname':~#_ ?
And then what is the exact command line you put?
For me no matter what is write the reply is: No such file or directory. I have put the install_container_manager.sh file into the /music folder...

**Reply by 007revad** [reply]
@Plantacom 
Try:
```
/volume1/music/install_container_manager.sh
```
or:
```
/volume1/music/ContainerManager_for_all_armv8/install_container_manager.sh
```

**Reply by Plantacom** [reply]
Hi. THX! Adding _/volume1_ to the line works. 
But have no clue (knowledge) why.... :)

**Comment by dantasan** [comment]
Working on DS119j (MARVELL Armada 3720 88F3720) with DSM 7.2.2-72806 Update 2
Thanks a lot for your script and work!
All the best from Austria!

**Comment by ThijsZijdel** [comment]
Working on DS218play @ DSM 7.2.1-69057 Update 6

<img width="1510" alt="dsm-update6" src="https://github.com/user-attachments/assets/474ce385-4dea-478e-9c18-191dbf374f1a" />

**Comment by nsenica** [comment]
Working great on DS218Play with DSM 7.2.2-72806 Update 3, the only caveat is if by any chance there's a power failure, Container Manager will not start, complains that it needs to be repaired and NAS needs to be rebooted for Container Manager to start properly. If there's anything I can provide to track this down, let me know.

**Reply by Tom8867** [reply]
Had the same issue after a power loss, had to uninstall and used the opportunity to install ContainerManager-armv8-24.0.2-1543, but after that I had to add my project again.

**Comment by christopherblack2012** [comment]
Working on DS418 with DSM 7.2.2-72806 Update 3. Took a few attempts to get it to work, but appears to work fine now. Thanks!

**Comment by Tom8867** [comment]
Working on Synology DS418
DSM Version: DSM 7.2.2-72806 Update 3
Container Manager: ContainerManager-armv8-24.0.2-15**43**, released June 20, 2025

**Comment by Tom8867** [comment]
**Update:**
Working on Synology DS418
DSM Version: DSM 7.**3**-81180
Container Manager: ContainerManager-armv8-24.0.2-**1606**, released September 25, 2025

**Comment by nsenica** [comment]
**Update:**
Working on Synology **DS218play**
DSM Version: DSM 7.3-81180
Container Manager: ContainerManager-armv8-24.0.2-1606

**Comment by magistersmax** [comment]
Update:
Working on Synology DS218
DSM Version: DSM 7.3-81180
Container Manager: ContainerManager-armv8-24.0.2-1606

**Reply by Dchris832** [reply]
I confirm

**Comment by drbartoli-crypto** [comment]
Thanks for the work, I have a problem, my DS418 was updated to DSM 7.2.2-72806 Update 4. I am trying to install the Container Manager and the script does not work. Any solution?

**Reply by Tom8867** [reply]
I think you have  to be a little bit more specific. "Does not work" could mean anything.
When you start the script, it "prepares" the Synology to accept that the downloaded Container Manager can be installed manually and after the manual installation, it sets the Synology intothe state like nothing happened.

**Comment by jgonza77** [comment]
Working on DS118 @ DSM Version: 7.3.2-86009
Container Manager: ContainerManager-armv8-24.0.2-1606

**Reply by MrMinisterMS** [reply]
I confirm

**Comment by Dchris832** [comment]
Working on DS218 @ DSM 7.3.2-86009 Update 3

**Comment by Tom8867** [comment]
Working on Synology DS418
DSM Version: DSM 7.**4.1**-90080
Container Manager: ContainerManager-armv8-24.0.2-**1706**, released July 31, 2026

**Reply by XanomalieX** [reply]
Did you just update and needed to run the script again to get the container manager?
Or did you prepare or uninstalled anything before the update?

**Reply by Tom8867** [reply]
I downloaded the latest Container manager, copied to my NAS and was running the script to update.
There was no need to uninstall the Container Manager.
For some weird reason, I can only use v2.0.9 of the script, any version higher than that, fails

Cheers

**Reply by XanomalieX** [reply]
I confirm

**Comment by speedking-in-rock** [comment]
Working on Synology DS218
DSM version: 7.4.1-90080
Container Manager: ContainerManager-armv8-24.0.2-1606

**Reply by speedking-in-rock** [reply]
Working on Synology DS218
DSM version: 7.4.1-90080
Container Manager: ContainerManager-armv8-24.0.2-**1706**
The script detected a the version available, but a manual update was required due to error "`Failed to query package list from server!`".

**Comment by nsenica** [comment]
Working on Synology DS218play
DSM version: 7.4.1-90080
Container Manager: ContainerManager-armv8-24.0.2-**1706**
